src/includes/getting-started-install/javascript.nextjs.mdx

```diff
@@ -0,0 +1,26 @@
+Add the `@sentry/nextjs` dependency:
+
+```npm
+npm install --save @sentry/nextjs
+```
+
+```yarn
+yarn add @sentry/nextjs
+```
+
+### Setup
+
+The call to the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) will automatically patch your project accordingly, though you can [setup manually](/platforms/javascript/guides/nextjs/manual-setup/) if you prefer.
+It will perform the following changes:
+
+- create `sentry.client.config.js` and `sentry.server.config.js` with the default `Sentry.init` (if they already exist, they will be `_sentry.client.config.js` and `_sentry.server.config.js` instead).
+- create `next.config.js` with the default configuration (if it already exists it will be `_next.config.js` instead, which will need to be merged manually).
+- create `sentry.properties` with Sentry’s project-specific data.
```

Comment:
Move this to replace the sentences that cover the same content under ## Configure. Editing here to trim:
```suggestion
<Note>

The call to the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) automatically patches your project, though you can [setup manually](/platforms/javascript/guides/nextjs/manual-setup/) if you prefer:

- creates `sentry.client.config.js`and `sentry.server.config.js` with the default `Sentry.init` (if they already exist, they will instead be appended with a preceding `_`).
- creates `next.config.js` with the default configuration (if it already exists, it will instead be appended with a preceding `_`, which will need to be merged manually).
- creates `sentry.properties` with Sentry’s project-specific data.

</Note>
```